test/Microsoft.Extensions.Caching.Memory.Tests/project.json

```diff
@@ -19,8 +19,8 @@
         "xunit.runner.console": "2.1.0"
       }
     },
-    "dnxcore50": { 
-      "imports": "portable-net45+win8",
+    "dnxcore50": {
+      "imports": "portable-net45+win8",         
```

Comment:
undo whitespace. Also, unrelated shouldn't net45 be net451? Same in other project.json